README.md

```diff
@@ -326,7 +326,7 @@ This repository also provides one such platforms where contributers come over an
 | [Tic-Tac-Toe Game](https://github.com/kunjgit/GameZone/tree/main/Games/Tic-Tac-Toe) |
 | [Rapid_click_frenzy](https://github.com/kunjgit/GameZone/tree/main/Games/Rapid_click_frenzy) 
 |[Penguins Can't Fly](https://github.com/Will2Jacks/GameZoneForked/tree/Task/Games/Penguins_Can't_Fly)|
```

Comment:
change the ' sign from the name and as well as from the link it is creating merge conflicts

Reply:
@ishita-43 I have not added that

Reply:
i  know but i was asking if you could resolve that .but its okay i will try other way!!